tox.ini

```diff
@@ -131,4 +133,4 @@ commands =
     pydocstyle {toxinidir}/kombu
 
 [testenv:mypy]
-commands = python -m mypy --config-file setup.cfg
+commands = python -m mypy --config-file setup.cfg
```

Comment:
you may want to configure your editor to automatically end a file with a linebreak ;)